modules/installation-methods-for-different-architectures.adoc

```diff
@@ -6,25 +6,18 @@
 = Installation methods for different architectures
 
 [role="_abstract"]
-{rh-rhacs-first} supports the following architectures.
+{rh-rhacs-first} supports the following architectures. For information on supported platforms and architecture, see the link:https://access.redhat.com/articles/7045053[Red Hat Advanced Cluster Security for Kubernetes Support Matrix].
 
-.Supported architectures and recommended installation methods
+.Architectures and recommended installation methods
```

Comment:
[nit] For ppc64le and s390x there is only one _available_ option, it's not a _recommendation_, at least as it seems to me.

```suggestion
.Supported installation methods per architecture
```